languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -56241,6 +56241,87 @@ though, and has since been slightly modified)-->
+                <antipattern>
+                    <token>c'</token>
+                    <token skip="2">est</token>
+                    <token postag_regexp="yes" postag="R (pers|dem) .*"/>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token>c'</token>
+                        <token skip="2">est</token>
+                    </marker>
+                    <token postag_regexp="yes" postag=".* p|sp .*"/>
+                </pattern>
+                <message>'C'est' suivi d'un pluriel devient 'ce sont'. Dans la langue parlée, on admet 'c'est' suivi par 'ceux/celles' ou 'eux|elles'</message>
+                <suggestion>ce sont</suggestion>
+                <example correction="Ce sont"><marker>C'est</marker> des barbares qui ont fait ça.</example>
+                <example>C'est eux qui ont eu l'idée.</example>
+                <example>C'est ceux qui ont eu l'idée.</example>
+            </rule>
+            <rule id="SE_BAFRER" name="se bâfrer">
+                <pattern>
+                    <marker>
+                        <token postag_regexp="yes" postag="R pers obj .*"/>
+                        <token postag_regexp="yes" postag="V etre .*" skip="1" min="0"/>
+                        <token postag_regexp="yes" postag="V .*" inflected="yes">bâfrer</token>
+                    </marker>
+                </pattern>
+                <message>'Bâfrer' ne prend pas de forme pronominale</message>
+                <suggestion><match no="2" postag="(V etre) (.*)" postag_regexp="yes" postag_replace="V avoir $2">avoir</match> \3</suggestion>
+                <example correction="a bâfré">>Il <marker>s'est bâfré</marker> au buffet à volonté.</example>
+                <example correction="bâfre">Il <marker>se bâfre</marker> au buffet à volonté.</example>
+            </rule>
+            <rule id="AUTREMENT_1" name="autrement plus">
+                <pattern>
+                    <marker>
+                        <token>autrement</token>
+                        <token>plus</token>
+                    </marker>
+                </pattern>
+                <message>'Autrement' étant déjà un comparatif au sens de 'bien plus', on se gardera de dire 'autrement plus' qui serait un pléonasme</message>
+                <suggestion>autrement</suggestion>
+                <example correction="autrement">Celui-ci est <marker>autrement plus</marker> travaillé que le précédent.</example>
+            </rule>
+            <rule id="AUTREMENT_2" name="autrement moins">
+                <pattern>
+                    <marker>
+                        <token>autrement</token>
+                        <token>moins</token>
+                    </marker>
+                </pattern>
+                <message>'Autrement' étant déjà un comparatif au sens de 'bien plus', on se gardera de dire 'autrement moins' qui serait un contresens</message>
```

Comment:
The message is missing a period (full stop) in the end.

Reply:
- [x]